src/point_one/fusion_engine/messages/solution.h

```diff
@@ -42,6 +42,9 @@ struct P1_ALIGNAS(4) PoseMessage : public MessagePayload {
   static constexpr uint8_t MESSAGE_VERSION = 1;
   static constexpr int16_t INVALID_UNDULATION = INT16_MIN;
 
+  /** Set this flag if the device is stationary. */
```

Comment:
Set ~this flag~ if ("this flag" sounds like something the user should do, not something that the device will do)